src/frontend/src/handler/create_table.rs

```diff
@@ -894,6 +895,7 @@ pub(super) async fn handle_create_table_plan(
     with_version_column: Option<String>,
     include_column_options: IncludeOption,
 ) -> Result<(PlanRef, Option<PbSource>, PbTable, TableJobType)> {
+    let col_id_gen = ColumnIdGenerator::new_initial();
```

Comment:
Move arg to eliminate unnecessary param

Reply:
Interesting. I believe we reused this function for `ALTER TABLE` prior to some refactoring and that's why we took an argument of column id generator. 🤡